### DISCUSSION TITLE
AWS Lambda Function.

### DISCUSSION THREAD

**Discussion by OwaisAzad** [EddieHubCommunity/support#7085] [Q&A] [answered]
What is the maximum execution time for AWS Lambda functions, and how can you increase this limit? Additionally, what are some strategies you can use to handle processing that requires more than the maximum execution time? DEFAULT ET = 15 minutes.

**Comment by shinyapes** [comment, marked as answer]
The maximum execution time for AWS Lambda functions is 900 seconds (15 minutes). This limit can be increased by contacting AWS support and requesting a limit increase.

To handle processing that requires more than the maximum execution time, there are several strategies that can be used:

Split the processing into smaller chunks: Break down the processing into smaller tasks that can be executed within the maximum execution time. Use AWS Step Functions to orchestrate the execution of these tasks.

Use AWS Batch: AWS Batch is a service that enables you to run batch computing workloads on the AWS Cloud. Use AWS Batch to run long-running processing tasks that require more than the maximum execution time.

Use AWS Glue: AWS Glue is a fully managed extract, transform, and load (ETL) service that makes it easy to move data between data stores. Use AWS Glue to process large datasets that require more than the maximum execution time.

Use AWS Fargate: AWS Fargate is a serverless compute engine for containers that makes it easy to run containers on AWS. Use AWS Fargate to run long-running processing tasks that require more than the maximum execution time.
